src/main/scala/com/databricks/labs/overwatch/utils/SparkSessionWrapper.scala

```diff
@@ -20,30 +33,38 @@ trait SparkSessionWrapper extends Serializable {
   @transient
   lazy protected val _envInit: Boolean = envInit()
 
+
+  protected def buildSpark(): SparkSession = {
+    sessionsMap.hashCode()
+    SparkSession
+      .builder()
+      .appName("GlobalSession")
+      .getOrCreate()
+  }
   /**
    * Access to spark
    * If testing locally or using DBConnect, the System variable "OVERWATCH" is set to "LOCAL" to make the code base
    * behavior differently to work in remote execution AND/OR local only mode but local only mode
    * requires some additional setup.
    */
-  lazy val spark: SparkSession = if (System.getenv("OVERWATCH") != "LOCAL") {
-    logger.log(Level.INFO, "Using Databricks SparkSession")
-    SparkSession
-      .builder().master("local").appName("OverwatchBatch")
-      .getOrCreate()
-  } else {
-    logger.log(Level.INFO, "Using Custom, local SparkSession")
-    SparkSession.builder()
-      .master("local[*]")
-      .config("spark.driver.maxResultSize", "8g")
-      .appName("OverwatchBatch")
-//    Useful configs for local spark configs and/or using labs/spark-local-execution
-//    https://github.com/databricks-academy/spark-local-execution
-//      .config("spark.driver.bindAddress", "0.0.0.0")
-//      .enableHiveSupport()
-//      .config("spark.warehouse.dir", "metastore")
-      .getOrCreate()
+  def spark(globalSession : Boolean = false): SparkSession = {
```

Comment:
This should be private[overwatch] -- I don't want people using this outside of the package, I want to ensure they use val spark

Reply:
done